src/services/ui/src/components/Opensearch/main/Table/index.tsx

```diff
@@ -76,13 +76,15 @@ export const OsTable: FC<{
         {/* TODO: Add a skeleton loader after discussing with HCD.
         See https://qmacbis.atlassian.net/browse/OY2-25623 */}
         {!context.data && (
-          <div className="p-4">
-            <LoadingSpinner />
-          </div>
+          <UI.TableRow>
+            <UI.TableCell>
+              <LoadingSpinner />
+            </UI.TableCell>
+          </UI.TableRow>
```

Comment:
This was throwing the 'div cannot be descendant of Table' error.
Wrapping it within a tr + td fixes it